docs/topics/migrations.txt

```diff
@@ -140,6 +140,13 @@ developers (or your production servers) check out the code, they'll
 get both the changes to your models and the accompanying migration at the
 same time.
 
+.. versionadded:: 1.8
```

Comment:
The docs about `--name` moved up from https://github.com/django/django/pull/3967/files#diff-4e7bc778c6cbf6f12bc8cebda76b5a8dL300 where they make much more sense.